Xamarin.Forms.Platform.Android/ExportRendererAttribute.cs

```diff
@@ -1,12 +1,15 @@
+using Android.OS;
 using System;
+using Xamarin.Forms.Core;
 
 namespace Xamarin.Forms
 {
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-	public sealed class ExportRendererAttribute : HandlerAttribute
+	public sealed class ExportRendererAttribute : BaseExportRendererAttribute
 	{
 		public ExportRendererAttribute(Type handler, Type target) : base(handler, target)
 		{
+			MajorVersion = (int)Build.VERSION.SdkInt;
```

Comment:
(Just realized) For Android, I opted in for SDK version (e.g. 26) as opposed to OS naming version (8 for Oreo). Let me know if this needs to be the other way around.